### PR TITLE
Added section on deprovision to upload KB and updated section for dep…

### PR DESCRIPTION
**Pull request by csterwa**
…rovision to be correct on migrate KB.

### CODE REVIEW

File: Ecosystem Partners/upload-service-manifests-to-addon-engine.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ The response from the `POST [base_url]` request should look similar to:
 }
 ```
 
-The Partner API response can provide any number of variables. For instance, a MySQL service might provide a response like:
+The provision response must include an `id` value that represents the service instnace unique identifier from the Partner's side. The Partner API response can provide any number of additional variables. For instance, a MySQL service might provide a response like:
```
Comment (dreverri) [inline Ecosystem Partners/upload-service-manifests-to-addon-engine.md]: typo: instnace

File: Ecosystem Partners/upload-service-manifests-to-addon-engine.md [hunk 2]
```diff
@@ -138,6 +138,21 @@ The Partner API response can provide any number of variables. For instance, a My
...
+The deprovision API endpoint using the same `[base_url]` as the provision API endpoint except that is called with the HTTP DELETE verb. Here is an example request:
+
+```
+DELETE [base_url]/:id
+```
+
+The `:id` that we send is the `id` that was returned in the provision response for the service instance. The expected successful response is:
+
+```
+200 ok
```
Comment (dreverri) [inline Ecosystem Partners/upload-service-manifests-to-addon-engine.md]: Uppercase `OK`?

File: Ecosystem Partners/upload-service-manifests-to-addon-engine.md [hunk 3]
```diff
@@ -138,6 +138,21 @@ The Partner API response can provide any number of variables. For instance, a My
 
 All of the attributes and their values returned in the provision response body are then provided to the consumer of the service instance.
 
+#### Deprovision API Endpoint
+
+The deprovision API endpoint using the same `[base_url]` as the provision API endpoint except that is called with the HTTP DELETE verb. Here is an example request:
```
Comment (dreverri) [inline Ecosystem Partners/upload-service-manifests-to-addon-engine.md]: "uses" instead of "using"?

Comment (dreverri) [inline Ecosystem Partners/upload-service-manifests-to-addon-engine.md]: "except that it is" instead of "except that is"?

**Comment by csterwa** [comment]
@dreverri thx! I made the corrections.

**Pull request merged**